apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -0,0 +1,216 @@
+import React, {PropTypes} from 'react';
+import {Button, FormControl} from 'react-bootstrap';
+import {Facilitator1819Program} from './detail_view_facilitator_specific_components';
+import $ from 'jquery';
+
+const renderLineItem = (key, value) => {
+  return value && (
+      <div>
+      <span style={{fontFamily: '"Gotham 7r"', marginRight: '10px'}}>
+        {key}:
+      </span>
+        {value}
+      </div>
+    );
+};
+
+class DetailViewContents extends React.Component {
+  static propTypes = {
+    applicationId: PropTypes.string.isRequired,
+    applicationData: PropTypes.shape({
+      regional_partner_name: PropTypes.string,
+      notes: PropTypes.string,
+      status: PropTypes.string.isRequired,
+      school_name: PropTypes.string,
+      district_name: PropTypes.string,
+      email: PropTypes.string,
+      formData: PropTypes.shape({
+        firstName: PropTypes.string.isRequired,
+        lastName: PropTypes.string.isRequired,
+        title: PropTypes.string,
+        phone: PropTypes.string,
+        preferredFirstName: PropTypes.string,
+        accountEmail: PropTypes.string,
+        alternateEmail: PropTypes.string,
+        program: PropTypes.string.isRequired,
+        planOnTeaching: PropTypes.arrayOf(PropTypes.string.isRequired),
+        abilityToMeetRequirements: PropTypes.string.isRequired,
+      })
+    }),
+  }
+
+  state = {
+    status: this.props.applicationData.status,
+    notes: this.props.applicationData.notes
+  }
+
+  handleCancelEditClick = () => {
+    this.setState({
+      editing: false,
+      status: this.props.applicationData.status
```

Comment:
is there a reason this is referring back to the props value and not the current state?

Reply:
yeah - we are resetting to initial values, not the current state.